src/main/resources/config.yml

```diff
@@ -119,6 +119,12 @@ hearts-on-death: true
 # If true, player will die directly when get damage from void
 die-on-void: false
 
+# Ressourcespawner spawns ressourcen inside of chests if it's located on spawner position
+# type: boolean
+# If true the Ressourcespawner spawns items inside a chest if it's on spawnlocation
+# If the chest is full, the items will drop naturally
+spawnRessourcesInChest: true
```

Comment:
We've always been using dash notation instead of camel-case in the config before. Should be changed.